content/blog/2017/2017-04-10-Jenkins-has-upgraded-to-Java-8.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Jenkins Now Requires Java 8 To Run Starting With 2.54"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+:imagesdir: /images/post-images/2017-04-java8
+
+image:lambda.png[role="right"] As link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced some weeks ago], starting with version link:https://jenkins.io/changelog/#v2.54[Jenkins **2.54**], the Java runtime requirement has been updated to Java 8.
+
+== Timeline
+
+* link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[Earlier this year: announce this upgrade].
+* April, 2017: Drop support for Java 7 in Jenkins weekly: **2.54**.
+* June, 2017: First LTS version requiring Java 8 is published.
+
+== FAQ
+
+I have only a JRE 7 available, can I still run Jenkins 2.54+ with it?::
+No, you need to install a JRE 8.
+
+Can I still run my agents with Java 7?::
+No. An agent must run at least the Java version which the Jenkins Master was compiled for.
+This means you must also upgrade your agents JVM if they are still running Java 7.
+
+== References
+
+* link:/blog/2016/11/22/what-jvm-versions-are-running-jenkins-the-return/[JVM statistics post back in November 2016]
+* link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[Official announcement on the Jenkins blog]
+* Original link:https://issues.jenkins-ci.org/browse/JENKINS-27624[JIRA ticket for this issue]
+* The link:https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jenkinsci-dev/fo5nKLhZK5U/Zb9jHpBJBQAJ[6 months, 82 messages, thread on the Jenkins developers mailing list]
+* The link:https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jenkinsci-users/R6SGnW7MY6A/25MBJ9a2CQAJ[announcement on the Jenkins users mailing list]
+* The link:https://github.com/jenkinsci/jenkins/pull/2802[Pull request on _Jenkins core_]
+
+As always, if you have questions or feedback about this announcement, please feel free to contact us.
```

Comment:
We do not want feedback on this announcement. Contributors and bug reporters only :)